### ISSUE TITLE
"If you liked this video then check out these other videos..."

### ISSUE THREAD

**Issue by unoukujou** [futo-org/grayjay-android#1182]
### Use Cases

I don't know exactly how to describe this but sometimes you watch a YouTube video and at the end of the video the guy says if you liked this video you should check out these other videos and he points to the corner of the screen but there's nothing there. Can this be included in grayjay?

**Comment by lmsalustri** [comment]
You're thinking of the INFO card set up by the creator

**Comment by unoukujou** [comment]
> You're thinking of the INFO card set up by the creator

Thanks. Any way these can be included in Grayjay? Sometimes they relate to the video you just watched and it feels like you're missing out by not seeing it.

If it can't be overlayed on top of the video then at least have it like a list of links in the bottom under the description. Maybe it's possible that way?

**Comment by SdoggaMan** [comment]
I think this is less so the info card and more the in-player overlay supported in YouTube clients. Most common at the end of a video during the outro where they'll throw up cards like the channel button and most recent/most popular or related content. (for example, go to the end of a Captain Disillusion video, where his graphics have an outline for each of those links [recommended/channel/patreon] and show nothing in Grayjay.) 

Personal opinion, I'm ambivalent to them, but they are I think a more natural and welcome part of the YouTube experience than, say, the full screen "WATCH THIS SHIT NEXT" present now in the native YouTube app, which is just recommended but designed to interrupt your ability to navigate away around the app and potentially close it. I'd suggest these be supported in Grayjay, if possible, for that reason, as they're A) set by the creator, so relevant to them, and B) can contain ways to support the creator further, such as watching more videos or clicking through to a Patreon.

**Comment by unoukujou** [comment]
Especially when those times when you're watching a video and the creator at some point in the middle of the video goes a bit off topic and says "I made another video in the past discussing this other topic so if you want to find out more about this feel free to check out that other video over here in the corner." But there's nothing there. You can only get this experience when watching the video on YouTube.

**Comment by SdoggaMan** [comment]
Agree. I guess in that regard--adding the overlaid interactible surfaces in native YouTube--the 'Info' card/popup and the annotations for video embeds/links are the same - both should be added, as, fundamentally, both are set by the creator and relevant to the content/creator and not just YouTube stay-on-my-platform-plz enshittification.

**Comment by mr-tin7** [comment]
Yeah, I need it too, but not just the ones at the end of video: sometimes in YouTube they also link videos on the top-right "i" (info tab), which is relevant to me because I watch videos about science topics which are interlinked to preliminary ones.
